src/hardware/vga_s3.cpp

```diff
@@ -446,15 +446,15 @@ void SVGA_S3_WriteSEQ(Bitu reg,Bitu val,Bitu iolen) {
 		vga.s3.pll.lock=val;
 		break;
 	case 0x10:		/* Memory PLL Data Low */
-		vga.s3.mclk.n=val & 0x1f;
-		vga.s3.mclk.r=val >> 5;
+		vga.s3.mclk.n = (val & 0b000'11111);
+		vga.s3.mclk.r = (val & 0b111'00000) >> 5;
```

Comment:
bit-notation is so immensely helpfully for me to see what bits are being passes through, as my mind doesn't automatically convert 0x1f to a bit-mask.